### ISSUE TITLE
Acme-challenge 404

### ISSUE THREAD

**Issue by truck0321** [JrCs/docker-letsencrypt-nginx-proxy-companion#166]
It looked like  #138 would have fixed this issue but it's still a problem for me.  The challenge file is created but if I attempt to go to that link Wordpress gives me a 404.  I've tried changing the vhost.d/default.conf mentioned in #138 but still get the same error.  

**Companion logs:**

`Sleep for 3600s
2017/02/12 06:20:41 Generated '/app/letsencrypt_service_data' from 3 containers
2017/02/12 06:20:41 Running '/app/update_certs'
2017/02/12 06:20:41 Watching docker events
Reloading nginx proxy...
2017/02/12 06:20:41 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/update_certs'
      h2017/02/12 06:20:41 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification ''
      ;2017/02/12 06:20:41 [notice] 33#33: signal process started
Creating/renewal  www.truckenmiller.com certificates... ( www.truckenmiller.com)
2017-02-12 06:20:41,475:INFO:simp_le:1211: Generating new account key
2017-02-12 06:20:41,847:INFO:requests.packages.urllib3.connectionpool:756: Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2017-02-12 06:20:42,572:INFO:requests.packages.urllib3.connectionpool:756: Starting new HTTPS connection (1): letsencrypt.org
2017-02-12 06:20:43,600:INFO:requests.packages.urllib3.connectionpool:207: Starting new HTTP connection (1): www.truckenmiller.com
2017-02-12 06:20:43,928:WARNING:simp_le:1303: www.truckenmiller.com was not successfully self-verified. CA is likely to fail as well!
2017-02-12 06:20:44,431:INFO:simp_le:1313: Generating new certificate private key
2017-02-12 06:20:53,367:ERROR:simp_le:1271: CA marked some of the authorizations as invalid, which likely means it could not access http://example.com/.well-known/acme-challenge/X. Did you set correct path in -d example.com:path or --default_root? Is there a warning log entry about unsuccessful self-verification? Are all your domains accessible from the internet? Failing authorizations: https://acme-v01.api.letsencrypt.org/acme/authz/NrV0M16U-98YVOTTZ-eOdOdZ1P88tNIgRehOgRfrBbo
Challenge validation has failed, see error log.`

**docker-compose:**

proxy:
  image: jwilder/nginx-proxy:latest
  restart: always
  ports:
    - "80:80"
    - "443:443"
  volumes: 
    - /var/run/docker.sock:/tmp/docker.sock:ro
    - /home/vagrant/vhost.d:/etc/nginx/vhost.d
    - /home/vagrant/certs:/etc/nginx/certs:ro
    - /home/vagrant/nginx/html:/usr/share/nginx/html
proxy-companion: 
  image: jrcs/letsencrypt-nginx-proxy-companion:v1.4
  restart: always
  volumes:
    - /home/vagrant/certs:/etc/nginx/certs:rw
    - /var/run/docker.sock:/var/run/docker.sock:ro
  volumes_from:
    - proxy

**Comment by truck0321** [comment]
I was editing the wrong file in vhost.d. There was a file named "default"  that I edited but once I made changes to the file named "www.truckenmiller.com"  I was able to access the challenge file.  However,  the CA is still saying it is invalid 

Here are the logs from the companion container:

Reloading nginx proxy...
2017/02/12 21:21:12 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/update_certs'
      h2017/02/12 21:21:12 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification ''
      ;2017/02/12 21:21:12 [notice] 33#33: signal process started
Creating/renewal  www.truckenmiller.com certificates... ( www.truckenmiller.com)
2017-02-12 21:21:12,588:INFO:simp_le:1211: Generating new account key
2017-02-12 21:21:12,961:INFO:requests.packages.urllib3.connectionpool:756: Starting new HTTPS connection (1): acme-v01.api.letsencrypt.org
2017-02-12 21:21:13,933:INFO:requests.packages.urllib3.connectionpool:756: Starting new HTTPS connection (1): letsencrypt.org
2017-02-12 21:21:14,828:INFO:requests.packages.urllib3.connectionpool:207: Starting new HTTP connection (1): www.truckenmiller.com
2017-02-12 21:21:14,852:INFO:simp_le:1305: www.truckenmiller.com was successfully self-verified
2017-02-12 21:21:14,982:INFO:simp_le:1313: Generating new certificate private key
2017-02-12 21:21:23,708:ERROR:simp_le:1271: CA marked some of the authorizations as invalid, which likely means it could not access http://example.com
Challenge validation has failed, see error log.
